Kentico.Kontent.Management/Modules/Extensions/UrlTemplateExtensions.cs

```diff
@@ -19,7 +19,7 @@ internal static string GetIdentifierUrlSegment(this UrlTemplate template, UserId
                 return string.Format(template.UrlEmail, identifier.Email);
             }
 
-            throw new ArgumentException("You must provide id, codename or externalId");
+            throw new ArgumentException("You must provide user id or email");
         }
 
```

Comment:
there are no tests for this class is this on purpose or there is no way to test this?

Reply:
Well, we don't have 100% coverage. We could add tests but it would be out of the scope of this PR.